KTL.js

```diff
@@ -5620,6 +5622,145 @@ function Ktl($, appInfo) {
+                    observeAndExecute(`#bulkOpsControlsDiv-${viewId}`, () => {
+                        ktl.views.stickTableColumns(viewId, numOfColumns, stickyColBkgdColor);
+                        isCalledInsideObserver = true;
+                        resolve();
+                    });
+                });
+
+                const callStickTableColumnsIfNotResolved = () => {
+                    if (!isCalledInsideObserver) {
+                        callStickTableColumns();
+                    }
+                };
+
+                // If observerPromise does not resolve within 1000ms, call callStickTableColumns
+                setTimeout(callStickTableColumnsIfNotResolved, 1000);
+            } else {
+                callStickTableColumns();
+            }
+        }
+
+        function isBulkOpsEnabled(keywords, viewId) {
+            if (keywords._nbo) return false;
+
+            const bulkOps = [
+                { operation: 'bulkEdit', role: 'Bulk Edit', class: 'cell-editable' },
+                { operation: 'bulkCopy', role: 'Bulk Copy', class: 'cell-editable' },
+                { operation: 'bulkDelete', role: 'Bulk Delete', selector: 'kn-delete-link' },
+            ];
+
+            return bulkOps.some(op => {
+                const isEnabled = ktl.core.getCfg().enabled.bulkOps[op.operation] && Knack.getUserRoleNames().includes(op.role);
+                if (!isEnabled) return false;
+                if (op.class) {
+                    ktl.core.waitSelector(`#${viewId} table.${op.class}`)
+                        .then(() => {
+                            return $(`#${viewId} table`).hasClass(op.class);
+                        });
+                }
+                if (op.selector) {
+                    return $(`#${viewId} .${op.selector}`).length > 0;
+                }
+                return true;
+            });
+        }
+
+        /**  Observe the Dom for a selector and execute a callback when the selector is found
+         * @param {string} selector - The selector to observe
+         * @param {function} callback - The callback to execute when the selector is found*/
+        function observeAndExecute(selector, callback) {
+            console.log('observeAndExecute:  ', selector);
+            const observer = new MutationObserver((mutationsList, observer) => {
+                for (let mutation of mutationsList) {
+                    if (mutation.addedNodes.length) {
+                        console.log('observeAndExecute:  ', mutation.addedNodes);
+                        const elementExists = $(mutation.target).find(selector).length > 0;
+                        if (elementExists) {
+                            callback();
+                            observer.disconnect();
+                            break;
+                        }
+                    }
+                }
+            });
+            observer.observe(document, { childList: true, subtree: true });
```

Comment:
@cortexrd  This has a major performance drawback, as I discussed with you. We should avoid this practice as much as possible.